tcl/dialog_startup.tcl

```diff
@@ -23,8 +23,6 @@ proc ::dialog_startup::chooseCommand { prompt initialValue } {
     wm minsize .inputbox 450 30
     wm resizable .inputbox 0 0
     wm geom .inputbox "450x30"
-    # not all Tcl/Tk versions or platforms support -topmost, so catch the error
-    catch {wm attributes $mytoplevel -topmost 1}
```

Comment:
(hmpf forgot to send my original post)

i don't really understand, why the `Preferences->Startup...` dialog does not have the `topmost` flag set, while Audio/Midi dialogs do.
what's the difference?

shouldn't we just fix the `$mytoplevel` instead?

~~~tcl
catch {wm attributes .inputbox -topmost 1}
~~~